src/app/children/previous-schools/previous-schools.ts

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This file should be removed because it is basically the same as `childSchoolRelation`